src/ImageSharp.Interactive/ImageSharp.Interactive.csproj

```diff
@@ -23,9 +23,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.DotNet.Interactive" Version="1.0.0-beta.21357.1" />
-    <PackageReference Include="Microsoft.DotNet.Interactive.Formatting" Version="1.0.0-beta.21357.1" />
+    <PackageReference Include="Microsoft.DotNet.Interactive" Version="1.0.0-beta.21404.3" />
+    <PackageReference Include="Microsoft.DotNet.Interactive.Formatting" Version="1.0.0-beta.21404.3" />
     <PackageReference Include="SixLabors.ImageSharp" Version="1.0.3" />
+    <PackageReference Include="SixLabors.ImageSharp.Drawing" Version="1.0.0-beta13" />
```

Comment:
I don't think we should add the beta just yet. The APIs are currently going through a major state of flux as we polish them up for the first RC.

Reply:
Good point, I will remove items looking for drawing support but I can get away with just image clad and little css, let me address that

Reply:
Done! I we merge can we release this package to nuget.org?

Reply:
https://github.com/SixLabors/ImageSharp.Interactive/pull/2/files#r684570700